bridge/hardhat.config.js

```diff
@@ -115,7 +135,8 @@ module.exports = {
     kovan: {
       live: true,
       url: 'wss://kovan.infura.io/ws/v3/' + INFURA_API_KEY,
```

Comment:
maybe we should take the chance to change all of the websockets URLs to HTTPS? since I think [it doesn't work with hardhat right now for custom connections](https://github.com/nomiclabs/hardhat/issues/1354)

Reply:
i agree

Reply:
done at https://github.com/rsksmart/tokenbridge/pull/269/commits/4068c20b90d789cd4164cdffe699095cb0a665a4